src/delta_time.rs

```diff
@@ -39,7 +39,7 @@ pub fn delta_time(year: f64) -> f64 {
     };
     let u = k + (year - 2000.0) / 100.0;
 
-    a0 + a1 * u.powi(1)
+    a0 + a1 * u
```

Comment:
`.powi(1)` was essentially a no-op, right?

Reply:
Indeed we can remove it, it was only there for the symmetry with the next operations but it's really not needed.